### PR TITLE
Channels Update

### PR DESCRIPTION
**Pull request by dirkkelly**
Includes the YouTube provider description which we'll make more of in the future.

### CODE REVIEW

File: data/channels/azurescapegoat.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ providers:
     name: azureScapegoat
     slug: UCXIN3HIH0eon5hVYYpnoNuQ
     url: https://www.youtube.com/channel/UCXIN3HIH0eon5hVYYpnoNuQ
-    description: ""
-    subscribers: 21950
+    description: Just your friendly neighborhood leftist cuck.
```
Comment (craiglonsdale) [inline data/channels/azurescapegoat.yml]: For consistency should this content be surrounded by quotation marks?

File: data/channels/badempanada.yml [hunk 2]
```diff
@@ -7,8 +7,9 @@ providers:
     name: BadEmpanada
     slug: UCUzmizB92LJ9oxf5T_snZNA
     url: https://www.youtube.com/channel/UCUzmizB92LJ9oxf5T_snZNA
-    description: ""
-    subscribers: 1581
+    description: Hi! I make well-researched videos about world history, politics,
```
Comment (craiglonsdale) [inline data/channels/badempanada.yml]: For consistency should this content be surrounded by quotation marks?

File: data/channels/badmouseproductions.yml [hunk 3]
```diff
@@ -6,8 +6,10 @@ providers:
     name: BadMouseProductions
     slug: xaxie1
     url: https://www.youtube.com/xaxie1
-    description: ""
-    subscribers: 64969
+    description: |-
```
Comment (craiglonsdale) [inline data/channels/badmouseproductions.yml]: Scuffed as symbols
also
For consistency should this content be surrounded by quotation marks?

File: data/channels/beauofthefifthcolumn.yml [hunk 4]
```diff
@@ -7,8 +7,9 @@ providers:
     name: Beau of the Fifth Column
     slug: UC0YvoAYGgdOfySQSLcxtu1w
     url: https://www.youtube.com/channel/UC0YvoAYGgdOfySQSLcxtu1w
-    description: ""
-    subscribers: 98331
+    description: The discussions of a southern journalist who is tired of a lack of
```
Comment (craiglonsdale) [inline data/channels/beauofthefifthcolumn.yml]: For consistency should this content be surrounded by quotation marks?

File: data/channels/benjiadamwhiskettes.yml [hunk 5]
```diff
@@ -6,8 +6,8 @@ providers:
     name: Benji Adam Whiskettes
     slug: UCDG87s_QtD4YrIamqb9k5WQ
     url: https://www.youtube.com/channel/UCDG87s_QtD4YrIamqb9k5WQ
-    description: ""
-    subscribers: 2922
+    description: So long and thanks for all the fish
```
Comment (craiglonsdale) [inline data/channels/benjiadamwhiskettes.yml]: For consistency should this content be surrounded by quotation marks?

File: data/channels/bigjoel.yml [hunk 6]
```diff
@@ -6,8 +6,9 @@ providers:
     name: Big Joel
     slug: UCaN8DZdc8EHo5y1LsQWMiig
     url: https://www.youtube.com/channel/UCaN8DZdc8EHo5y1LsQWMiig
-    description: ""
-    subscribers: 151117
+    description: I make video essays and deep dives about movies and other things.
```
Comment (craiglonsdale) [inline data/channels/bigjoel.yml]: For consistency should this content be surrounded by quotation marks?

File: data/channels/blackgoat666.yml [hunk 7]
```diff
@@ -7,7 +7,7 @@ providers:
     slug: UCbbsW7_Esx8QZ8PgJ13pGxw
     url: https://www.youtube.com/channel/UCbbsW7_Esx8QZ8PgJ13pGxw
     description: ""
```
Comment (craiglonsdale) [inline data/channels/blackgoat666.yml]: Purposefully left empty?

File: data/channels/bronxblogger.yml [hunk 8]
```diff
@@ -6,8 +6,8 @@ providers:
     name: the Bronx Blogger
     slug: UCxkk0VDNt0OJ3AZMNlRrGNg
     url: https://www.youtube.com/channel/UCxkk0VDNt0OJ3AZMNlRrGNg
-    description: ""
-    subscribers: 1429
+    description: '"Justice is what love looks like in public" -Dr. Cornel West'
```
Comment (craiglonsdale) [inline data/channels/bronxblogger.yml]: '". should maybe be "', or "'" lol i don't know

File: data/channels/c0nc0rdance.yml [hunk 9]
```diff
@@ -7,8 +7,17 @@ providers:
     name: C0nc0rdance
     slug: C0nc0rdance
     url: https://youtube.com/C0nc0rdance
-    description: ""
-    subscribers: 54153
+    description: |-
```
Comment (craiglonsdale) [inline data/channels/c0nc0rdance.yml]: scuffed as symbols
also
For consistency should this content be surrounded by quotation marks?

**Comment by dirkkelly** [comment]
Good questions Craig. All a side effect of the yaml library we’re using in bake. @j16r has recommended another one. Hopefully we can have all descriptions output on new lines without the quote issues. Will merge this later today after repudiating the content to fix merge issues.

**Pull request merged**